addon/manifest-base.json

```diff
@@ -0,0 +1,39 @@
+{
+  "manifest_version": 2,
+  "name": "Test Pilot",
+  "author": "Mozilla (https://mozilla.org/)",
+  "applications": {
+    "gecko": {
+      "id": "@testpilot-addon",

```

Comment:
Might want to change the ID in `process-with-env` for the add-on based on environment, too. Maybe append `-$ENVIRONMENT_TITLE` in builds except for `production`? That could help keep us from clobbering one environment's add-on with another